javascript-practice/src/views/pages/add-product.html

```diff
@@ -0,0 +1,44 @@
+<section>
+  <div class="container add-product-container">
+    <h1 class="main-heading">Add New Product</h1>
+    <form action="javascript:void(0)" class="form-default add-form"
+    enctype="multipart/form-data">
+      <div class="flex-column">
+        <label class="label-primary" for="name">Name</label>
+        <input type="text" class="form-control input-size-md">
+      </div>
+      <div class="flex-column">
+        <label class="label-primary" for="brand">Brand</label>
+        <input type="text" class="form-control input-size-md">
+      </div>
+      <div class="flex-column">
+        <label class="label-primary" for="model">Model Name</label>
+        <input type="text" class="form-control input-size-md">
+      </div>
+      <div class="flex-column">
+        <label class="label-primary" for="color">Color</label>
+        <input type="text" class="form-control input-size-md">
+      </div>
+      <div class="flex-column">
+        <label class="label-primary" for="form-factor">Form Factor</label>
+        <input type="text" class="form-control input-size-md">
+      </div>
+      <div class="flex-column">
+        <label class="label-primary" for="connectivity-technology">Connectivity Technology</label>
+        <input type="text" class="form-control input-size-md">
+      </div>
+      <div class="flex-column">
+        <label class="label-primary" for="amount">Amount</label>
+        <input type="text" class="form-control input-size-md">
+      </div>
+      <div class="flex-column">
+        <label class="label-primary" for="product-image">Image</label>
+        <input type="file" id="product-image" class="form-control input-size-md">
+      </div>
+      <div class="flex-row">
+        <button type="submit" class="btn btn-primary btn-submit">Submit</button>
+        <button type="reset" class="btn btn-primary btn-danger">Reset</button>
```

Comment:
Should be a cancel button instead of a Reset button. We'll back to product list page when user click Cancel